diff --git a/node_modules/antd/es/list/Item.d.ts b/node_modules/antd/es/list/Item.d.ts
index 9baddea..73552e6 100644
--- a/node_modules/antd/es/list/Item.d.ts
+++ b/node_modules/antd/es/list/Item.d.ts
@@ -1,4 +1,4 @@
-import type { CSSProperties, FC, ForwardRefExoticComponent, HTMLAttributes, ReactNode } from 'react';
+import { CSSProperties, FC, ForwardRefExoticComponent, HTMLAttributes, ReactNode } from 'react';
 import React from 'react';
 export interface ListItemProps extends HTMLAttributes<HTMLDivElement> {
     className?: string;
diff --git a/node_modules/antd/es/list/index.d.ts b/node_modules/antd/es/list/index.d.ts
index b1e3652..55ea536 100644
--- a/node_modules/antd/es/list/index.d.ts
+++ b/node_modules/antd/es/list/index.d.ts
@@ -1,8 +1,8 @@
 import * as React from 'react';
-import type { PaginationConfig } from '../pagination';
-import type { SpinProps } from '../spin';
-export type { ListItemMetaProps, ListItemProps } from './Item';
-export type { ListConsumerProps } from './context';
+import { PaginationConfig } from '../pagination';
+import { SpinProps } from '../spin';
+export { ListItemMetaProps, ListItemProps } from './Item';
+export { ListConsumerProps } from './context';
 export type ColumnCount = number;
 export type ColumnType = 'gutter' | 'column' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
 export interface ListGridType {
